src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -136,6 +136,16 @@ public EmailWrapper generateFeedbackSessionSummaryOfCourse(String courseId, Stud
                         .toAbsoluteString();
                 submitUrlHtml = "<a href=\"" + submitUrl + "\">" + submitUrl + "</a>";
             }
+            
+            if (fsa.isClosed()) {
+                String viewUrl = Config.getAppUrl(Const.ActionURIs.STUDENT_FEEDBACK_SUBMISSION_EDIT_SAVE)
```

Comment:
Two issues:
- The base URL is wrong: should be EDIT_**PAGE** rather than EDIT_**SAVE**.
- It looks identical with the `if` block before this. Why not just change the conditional?